hack/local-up-cluster.sh

```diff
@@ -842,6 +847,11 @@ if [[ "${START_MODE}" != "nokubelet" ]]; then
   start_kubelet
 fi
 
+START_DISCOVERY=${START_DISCOVERY:-false}
+if [[ "${START_DISCOVERY}" = true ]]; then
```

Comment:
@xilabao I've defaulted this to off.  I'm reworking it to run as a pod in a future pull.  I'll tag you in it.